pkg/operator/controller/ingress/load_balancer_service.go

```diff
@@ -260,10 +260,16 @@ func (r *reconciler) ensureLoadBalancerService(ci *operatorv1.IngressController,
 		return false, nil, err
 	}
 
+	// BZ2054200: Don't modify/delete services that are not directly owned by this controller
```

Comment:
```suggestion
	// BZ2054200: Don't modify/delete services that are not directly owned by this controller.
```
(Sorry, I know I'm being picky.)

Reply:
Done and in nodeport_service.go as well.